web_src/js/features/common-issue.js

```diff
@@ -58,16 +57,9 @@ export function initCommonIssue() {
       // checkboxes stay checked after reload
       if (action === 'close' || action === 'open') {
         // uncheck all checkboxes
-        $('.issue-checkbox input[type="checkbox"]').each((_, e) => { e.checked = false });
+        $('.issue-checkbox').each((_, e) => { e.checked = false });
```

Comment:
It might be that this unchecking is also no longer necessary, will verify.

Reply:
Added `autocomplete=off` attribute to the checkboxes and removed these hacky JS blocks. Outcome is the same, no state caching between page reloads.